qa/wildfly-runtime/src/main/wildfly/standalone/configuration/standalone.xml

```diff
@@ -591,16 +596,16 @@
             <inet-address value="${jboss.bind.address:127.0.0.1}"/>
         </interface>
     </interfaces>
-    <socket-binding-group name="standard-sockets" default-interface="public" port-offset="0">
-        <socket-binding name="ajp" port="${jboss.ajp.port}"/>
-        <socket-binding name="http" port="${jboss.http.port}"/>
-        <socket-binding name="https" port="${jboss.https.port}"/>
-        <socket-binding name="management-http" interface="management" port="${jboss.management-http.port}"/>
-        <socket-binding name="management-https" interface="management" port="${jboss.management-https.port}"/>
-        <socket-binding name="txn-recovery-environment" port="${jboss.txn-recovery-environment.port}"/>
-        <socket-binding name="txn-status-manager" port="${jboss.txn-status-manager.port}"/>
+    <socket-binding-group name="standard-sockets" default-interface="public" port-offset="${jboss.socket.binding.port-offset:0}">
+        <socket-binding name="ajp" port="${jboss.ajp.port:8009}"/>
+        <socket-binding name="http" port="${jboss.http.port:8080}"/>
+        <socket-binding name="https" port="${jboss.https.port:8443}"/>
+        <socket-binding name="management-http" interface="management" port="${jboss.management.http.port:9990}"/>
+        <socket-binding name="management-https" interface="management" port="${jboss.management.https.port:9993}"/>
+        <socket-binding name="txn-recovery-environment" port="4712"/>
+        <socket-binding name="txn-status-manager" port="4713"/>
         <outbound-socket-binding name="mail-smtp">
-            <remote-destination host="localhost" port="${jboss.mail-smtp.port}"/>
+            <remote-destination host="localhost" port="25"/>
```

Comment:
🔧 Try to keep the old values where possible, Arquillian might not be able to reach the server anymore otherwise.